lib/workers/branch/lock-files.js

```diff
@@ -123,6 +123,34 @@ async function writeExistingFiles(config) {
         upath.join(basedir, 'package.json'),
         JSON.stringify(massagedFile)
       );
+
+      // if you have some reference to local files, copy them
+      const toCopy = listLocalLibs(massagedFile.dependencies);
+      toCopy.push(...listLocalLibs(massagedFile.devDependencies));
+      if (toCopy.length !== 0) {
+        logger.debug(`listOfNeededLocalFiles files to copy: ${toCopy}`);
+        await Promise.all(
+          toCopy.map(async localPath => {
+            const resolvedLocalPath = path.resolve(basedir, localPath);
+            // at the root of local Lib we should find a package.json and yarn will use it to update yarn.lock file
+            const resolvedRepoPath = upath.join(
+              resolvedLocalPath.substring(config.tmpDir.path.length + 1),
+              'package.json'
+            );
+            const fileContent = await platform.getFile(resolvedRepoPath);
+            if (fileContent !== null) {
+              await fs.outputFile(
+                upath.join(resolvedLocalPath, 'package.json'),
+                fileContent
+              );
+            } else {
+              logger.error(
```

Comment:
Should be user error, not server-side. We can perhaps make it a warn for now to help us check for any bugs but once this functionality seems solid it should be converted to an info message on server side.

Reply:
got it